src/main/java/com/example/Main.java

```diff
@@ -24,7 +24,30 @@ public static HttpServer startServer() {
         // create a resource config that scans for JAX-RS resources and providers
         // in com.example package
         final ResourceConfig rc = new ResourceConfig().packages("com.example");
-
+        
+        
+        
+        // querying the database for a customer list
+        String query = "SELECT Name, PostalAddress, ZipCode, BankAccountNo " +
+        "FROM Customer " +
+        "WHERE CustomerId =" + CustomerId;
+        
+        ResultSet rs = executeQuery(query)
+        
```

Comment:
Could you please tell me why you are also querying for our customer's bank account number? I believe it is wrong to expose this kind of sensitive information to our website's administrators. It could be potentially a big security breach and against our SLA.

You should modify your query to something like: "SELECT Name, PostalAddress, ZipCode"